### ISSUE TITLE
GIF for README

### ISSUE THREAD

**Issue by celestelayne** [celestelayne/tipcalculator#2]
![tippr](https://cloud.githubusercontent.com/assets/5299069/5795604/5eb9ff42-9f48-11e4-86e0-45b4acfc9711.gif)

**Comment by timothy1ee** [comment]
Good work. This prework is a preview of our weekly project process. Generally, weekly projects take between 5-10 hours to complete the required features and an additional 5-10 hours to complete the optional features. In general, we've seen that the more hours you log, the quicker you improve your proficiency with iOS.

It doesn't look like you meet our requirements for the designer class which involves a design-related degree (or equivalent experience) in addition to 2+ years of design experience. See more details [here](https://courses.codepath.com/snippets/ios_for_designers/thanks_for_applying#heading-requirements).

In the meantime, you're welcome to follow along with the course remotely. If you like, I can add you as a course observer. As a course observer, you have access to the recorded class lecture, the course materials, and you can follow along with the weekly projects.

**Comment by celestelayne** [comment]
/cc @codepathreview Hi, thanks Tim,

I'm confused. You say that I don't have a design-related degree (or equivalent experience) in addition to 2+ years of design experience. 

I attended PennDesign graduate program at the university of Pennsylvania and worked as an architectural designer for 5 years.

Are you looking fora specific type of design degree?

**Comment by timothy1ee** [comment]
Yes, sorry, we should clarify that in the application. This class is for web / mobile designers who are interested in learning how to prototype their designs.

**Comment by celestelayne** [comment]
ok, thank you. i really enjoyed making this and would love to follow along with the course remotely. Your instructions were very clear. Can you add me as a course observer?

**Comment by timothy1ee** [comment]
I've added you as a course observer.

**Comment by celestelayne** [comment]
Thanks!
